pkg/controllers/spannerautoscaler_controller.go

```diff
@@ -105,7 +105,14 @@ func NewSpannerAutoscalerReconciler(
 	scheme *runtime.Scheme,
 	recorder record.EventRecorder,
 	opts ...Option,
-) *SpannerAutoscalerReconciler {
+) (*SpannerAutoscalerReconciler, error) {
+	config := zap.NewProductionConfig()
```

Comment:
I think it is better to construct own logger rather than using `NewProductionConfig`.
My recommended setting is 

```
	config := zap.Config{
		Level:             zap.NewAtomicLevelAt(zap.InfoLevel),
		Development:       false,
		Encoding:          "json",
		DisableCaller:     true,
		DisableStacktrace: true,
		EncoderConfig: zapcore.EncoderConfig{
			TimeKey:        "timestamp",
			LevelKey:       "level",
			NameKey:        "logger",
			MessageKey:     "message",
			LineEnding:     zapcore.DefaultLineEnding,
			EncodeLevel:    zapcore.LowercaseLevelEncoder,
			EncodeTime:     zapcore.EpochMillisTimeEncoder,
			EncodeDuration: zapcore.SecondsDurationEncoder,
		},
		OutputPaths:      []string{"stdout"},
		ErrorOutputPaths: []string{"stderr"},
	}
```

side note:
To use Datadog Log, we have to change `EncodeTime` to `EpochMillisTimeEncode`.

Reply:
Fixed!